app/support/reservations/validations.rb

```diff
@@ -81,6 +82,27 @@ def conflicting_reservation(options = {})
     conflicting_reservations.first
   end
 
+  def does_not_conflict_with_admin_reservation?
+    conflicting_admin_reservation.nil?
+  end
+
+  def does_not_conflict_with_admin_reservation
+    errors.add(:base, :conflict) if conflicting_admin_reservation
+  end
+
+  def conflicting_admin_reservation
+    conflicting_reservations =
+      Reservation
```

Comment:
Part of me wants to DRY up this scope, but I'm not sure if it's worth it.

Reply:
I'm gonna vote for not worth it right now.